test/functional/ui/tabline_spec.lua

```diff
@@ -184,4 +184,29 @@ describe('tabline', function()
     ]],
     }
   end)
+
+  it('middle closes tab', function()
+    insert('tab1')
+    command('tabnew')
+    insert('tab2')
+    command('tabprev')
+    api.nvim_set_option_value('tabline', '%1T口口%2Ta' .. ('b'):rep(38) .. '%999Xc', {})
```

Comment:
is this relevant to this test?

Reply:
It is not, I believe I found a better way to test this.